index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
+    <link rel="icon" type="image/png" href="/public/favicon.png" />
```

Comment:
It's looks better like
```suggestion
    <link rel="icon" type="image/png" href="/favicon.png" />
```

Reply:
Can you do this? Is there any extra config we need to do so this can actually work?

Reply:
Yes, it's default behavior. Equally, I tested and works!